src/app/games/game-add/game-add.component.ts

```diff
@@ -19,46 +23,45 @@ import { LocationsService } from '../locations.service';
 })
 export class GameAddComponent {
 
+    get searchDebounceTime(): number { return 750; }
+    get noTeam(): Team {
+        return {
+            id: -1,
```

Comment:
I'd prefer `null` or `undefined`. Boolean for -1 is true in both JavaScript and Python. Also, it's better semantically